client/components/tinymce/plugins/mentions/editor-mention.jsx

```diff
@@ -3,18 +3,12 @@
  */
 import React from 'react';
 
-export default class EditorMention extends React.Component {
-	static propTypes = {
-		username: React.PropTypes.string
-	};
+const EditorMention = ( { username } ) => (
+	<span>@{ username }</span>
+);
 
-	render() {
-		const { username } = this.props;
+EditorMention.propTypes = {
+	username: React.PropTypes.string,
```

Comment:
this is probably required, no? we could add `React.PropTypes.string.isRequired`

the reason behind this is that a username with no text isn't a username…